src/classes/TDTM_Filter_TEST.cls

```diff
@@ -74,7 +74,7 @@ public with sharing class TDTM_Filter_TEST {
                 token.filterValue = null;
             }
 		}
-        TDTM_Global_API.setTdtmConfig(tokens);
+        TDTM_Global_API.setTdtmConfig(tokens, 'hed');
```

Comment:
Actually, the namespace 'hed' here is just a way to differentiate HEDA default configs and other configs. It is not use to differentiate the managed and unmanaged environment. Also, the handlers in TDTM_DefaultConfig class all have the Owned_by_Namespace__c = 'hed'. So, the test cases will pass in the unmanaged environment. 

In the existing test cases, the  TDTM_Global_API.setTdtmConfig(tokens) also works since there is no trigger handler in the database yet. So, TDTM_Global_API.setTdtmConfig(tokens) and TDTM_Global_API.setTdtmConfig(tokens, 'hed') will return you the same result. 

Theoretically, user should use TDTM_Global_API.setTdtmConfig(tokens, 'hed') to manipulate HEDA tokens. And use TDTM_Global_API.setTdtmConfig(tokens) to manipulate tokens without namespace. However, I can reserve the change for existing test cases since the behavior is the same.